doc/users/prev_whats_new/github_stats_3.3.0.rst

```diff
@@ -918,7 +918,7 @@ Pull Requests (1066):
 * :ghpull:`16277`: Prefer using MouseButton to numeric values in docs and defaults.
 * :ghpull:`16270`: numpydoc-ify SymLogNorm
 * :ghpull:`16274`: Tiny cleanups to set_xlabel(..., loc=...).
-* :ghpull:`16273`: DOC: Changing the spelling of co-ordinates.
+* :ghpull:`16273`: DOC: Changing the spelling of coordinates.
```

Comment:
Except for this one; the PR was updating the spelling, so the original is correct there.

Reply:
Resolved